Extensions/StyleCop/StyleCopTask/StyleCop.ps1

```diff
@@ -2,6 +2,7 @@ param
 (
     [string]$treatStyleCopViolationsErrorsAsWarnings,
     [string]$maximumViolationCount,
+	[string]$allowableViolationCount,
```

Comment:
I can do but looking at the change it actually fixes missing indentation so might be preferential (this was IDE added, I hadn't noticed).